### PR TITLE
Reference app-of-apps from argocd-apps repo.

### PR DESCRIPTION
**Pull request by HumairAK**
Dependent on: https://github.com/operate-first/argocd-apps/pull/119
Related: https://github.com/operate-first/argocd-apps/issues/115

### CODE REVIEW

**Comment by HumairAK** [comment]
(the test will fail until the dependent pr is merged)

File: manifests/overlays/moc-infra/applications/kustomization.yaml [hunk 1]
```diff
@@ -1,6 +1,4 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-  - app-of-apps-curator.yaml
-  - app-of-apps-infra.yaml
-  - app-of-apps-zero.yaml
+  - github.com/operate-first/argocd-apps/app-of-apps?ref=master
```
Comment (tumido) [inline manifests/overlays/moc-infra/applications/kustomization.yaml]: FTR, added to the list here https://github.com/operate-first/blueprint/issues/50

**Review by tumido — COMMENTED** [review]
/lgtm

File: manifests/overlays/moc-infra/applications/kustomization.yaml (hunk 1 above)
Comment (HumairAK) [reply manifests/overlays/moc-infra/applications/kustomization.yaml]: it should actually be `main` woops

**Review by tumido — COMMENTED** [review]
/lgtm

**Pull request merged**